src/anemoi/inference/outputs/gribfile.py

```diff
@@ -156,7 +161,10 @@ def __init__(
         variables : list, optional
             The list of variables, by default None.
         split_output : bool, optional
-            Whether to split the output, by default True.
+            Whether to split the output, by default None.
+            Cannot be `True` if `out` is a file-like object.
+            If `out` is a string, this is set to `True` by default.
+            If `out` is a file-like object, this is ignored and set to `False`.
```

Comment:
Delete these last 2 lines? Just `Cannot be True if out is a file-like object.` is enough?